.github/ISSUE_TEMPLATE/custom_product/tech-day.md

```diff
@@ -7,36 +7,52 @@ labels: ['Tech Day', 'Area:Tech']
-- Work on items identified during the first hour.
-- Wrap up your day by spending 1 hour to document the changes you did, provide instructions for testing and eventually to give pointers to the next person working on a techday ticket for this codebase.
+You will not be able to address the entierety of a codebase tech debt in one go, organization is key to make sure you wrap-up time spent on a codebase with concrete deliverables.
+
+When dedicating a fixed amount of time on a codebase, we recommend the following schedule:
+- Begin by reviewing the checklist attached to this ticket, identify and prioritize activities you would be able to complete within the dedicated time.
+- Work on said items.
+- Wrap up by briefly documenting the changes you did, provide instructions for testing and eventually create issues for activities you'd like to work on next. You can do so by adding a comment to this issue.
 
 ## Create tickets for future work
-Not all tech debt items can be addressed within a day, the goal of this day is also to raise awareness about tech debt to be tackled in the future.
+Not all tech debt items can be addressed in one day, one of the goal of the ownership initiative is also to raise awareness about tech debt to be tackled in the future.
 
-If you see a non-compliant element but you didn't get a chance to work on it, please create the corresponding ticket, attach it to the next fixVersion of the codebase and link it in this ticket.
+When creating such tickets, try to provide details about complexity of such an implementation. These elements play a role in our capacity to prioritize work.
 
-Please fill the checklist available in this ticket, priorities are available as a guideline as to what we consider more or less important for each of the tech areas:
+## About priorities
 
-🚨 Indicates a required item, to be looked at during the day
-🔝 Indicates a top priority item
-🟠 Indicates a medium priority item
-🙏 Indicates a low priority item
+* 🚨 Indicates a required item, to be looked at
+* 🔝 Indicates a top priority item
+* 🟠 Indicates a medium priority item
+* 🙏 Indicates a low priority item
 
-## Tech day checklist
+## Checklist
 
-This checklist is focused on a classic Jahia repository (module, app)
+This checklist is there to help you but is not exaustive, if some items are not relevant or should be added, [please request a change](https://github.com/Jahia/.github/blob/master/.github/ISSUE_TEMPLATE/custom_product/tech-day.md).
 
 ### General
-- [ ] 🚨 I reviewed all OPEN TECH tickets created for that codebase (using fixVersion=codebase-X.Y.Z-SNAPSHOT)
+- [ ] 🚨 I reviewed all OPEN TECH tickets created for that codebase (using codebase-X.Y.Z milestone)
 - [ ] 🚨 I reviewed older tech day tickets for that codebase
+- [ ] 🟠 Standards have been discussed in a tech kumite in the past semester
```

Comment:
This needs a link to a summary of those standards. If not possible to provide it, this entry does not add that much information and can be removed,